### PR TITLE
Moody 60 add image value type

### PR DESCRIPTION
**Pull request by AhmetKuris**
Tests are coming. Could you please first check their bare code?

### CODE REVIEW

File: LetsDoIt.CustomValueTypes/src/LetsDoIt.CustomValueTypes/Image/ImageHelper.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Drawing;
+using System.IO;
+
+namespace LetsDoIt.CustomValueTypes.Image
+{
+    public static class ImageHelper
+    {
+        public static string Resize(byte[] image, int width, int height)
```
Comment (durnasaban) [inline LetsDoIt.CustomValueTypes/src/LetsDoIt.CustomValueTypes/Image/ImageHelper.cs]: We can make it an extension method

File: LetsDoIt.CustomValueTypes/test/LetsDoIt.CustomValueTypes.Tests/EmailTests.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    public class EmailTests
+    {
+
+        [Theory]
+        [InlineData(null)]
+        [InlineData("")]
+        public void Email_WhenIsNullOrWhiteSpace_ShouldThrowArgumentException(string email)
+        {
+           void Test() => new Email(email);
```
Comment (durnasaban) [inline LetsDoIt.CustomValueTypes/test/LetsDoIt.CustomValueTypes.Tests/EmailTests.cs:19]: We can improve this test with fluent assertion

**Review by durnasaban — CHANGES_REQUESTED** [review]

File: LetsDoIt.CustomValueTypes/src/LetsDoIt.CustomValueTypes/Image/ImageHelper.cs (hunk 1 above)
Comment (AhmetKuris) [reply LetsDoIt.CustomValueTypes/src/LetsDoIt.CustomValueTypes/Image/ImageHelper.cs]: Done.

File: LetsDoIt.CustomValueTypes/test/LetsDoIt.CustomValueTypes.Tests/EmailTests.cs (hunk 2 above)
Comment (AhmetKuris) [reply LetsDoIt.CustomValueTypes/test/LetsDoIt.CustomValueTypes.Tests/EmailTests.cs:19]: Done

**Review by durnasaban — APPROVED** [review]

**Pull request merged**